hibike/devices/YogiBear/YogiBear.cpp

```diff
@@ -24,8 +24,9 @@ void setup() {
   PIDSetup();
   setup_LEDs();
   test_LEDs();
-  hibike_setup(500); // Time in milliseconds before timeout on heartbeat
-
+  hibike_setup();
```

Comment:
you want to keep the 500 arg. 

When rebasing, have a preference for keeping master rev over local rev. 
If you cant tell, you should ask someone, because its very likely that reverting master changes also revert functionality.